src/Resources/Deals.php

```diff
@@ -16,11 +16,21 @@ class Deals extends Resource
      *
      * @return \SevenShores\Hubspot\Http\Response
      */
-    public function create(array $properties)
+    public function create(array $properties, array $associations = [])
```

Comment:
Please add `associations` param to method signature https://github.com/HubSpot/hubspot-php/pull/332/files#diff-7d4670c87d7bdd8120b7c0cb12cf3aa4R13